### DISCUSSION TITLE
Definition question in DoPolarizationSums function?

### DISCUSSION THREAD

**Discussion by rmastand** [FeynCalc/feyncalc#129] [General]
In the DoPolarizationSums function (defined [here](https://feyncalc.github.io/FeynCalcBook/ref/DoPolarizationSums.html)), there's a reference to an "auxiliary 4-vector from the gauge-dependent polarization sum formula" n. What 4-vector is this referring to? I can only find this term defined in the FeynCalc reference manual, so I can't cross check it with another QFT resource.

Thank you!

**Comment by vsht** [comment]
The 4-vector is arbitrary as long as it doesn't give zero when contracted with the
4-momentum of the respective vector boson. I think that every QFT reference that 
discusses how to square QCD matrix elements in some details should cover that part, e.g.

- p.45 in https://cds.cern.ch/record/454171/files/p41.pdf
- Eq. A.1.46 in Böhm, Denner, Joos, "Gauge theories of the Strong Electroweak Interactions"
